File: Pacman-Game-OOP-master 2/EASendMail/Samples_ASPNetMvc/C#/WebProject1/WebProject1/Services/SendMailThreadState.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using EASendMail;

namespace WebProject1.Services
{
    class SendMailThreadState
    {
        public SmtpMail Mail { get; set; } = null;
        public SmtpServer Server { get; set; } = null;
        public int RecipientIndex { get; set; } = 0;
        public string TaskId { get; set; } = string.Empty;
        public int  DbTaskId { get; set; } = 0;

    }
}

File: Pacman-Game-OOP-master 2/EASendMail/Samples_ASPNetMvc/VB/WebProject1/WebProject1/Services/SendMailThreadState.vb
Imports EASendMail

Class SendMailThreadState
    Public Property Mail As SmtpMail = Nothing
    Public Property Server As SmtpServer = Nothing
    Public Property RecipientIndex As Integer = 0
    Public Property TaskId As String = String.Empty
    Public Property DbTaskId As Integer = 0
End Class
